### PR TITLE
[MIRROR] Adds Armolitskiy's and Eluxor's sprites and items at Reere's Request

### PR DESCRIPTION
**Pull request by drexample**
Original PR: https://github.com/discordia-space/CEV-Eris/pull/5019
--------------------
## About The Pull Request

Adds different sprites for and a cape to TE's hardsuit, adds a jacket for the TE, adds Technomancer armor, adds a bomj coat to random loot and loadout, and updates the sprites of the Technomancer and TE jumpsuits. Hardsuits now show modules on the person wearing them if the modules are set to, may cause unforseen consequences far as I know. Sprites by Armolitskiy (Armolitskiy#4866 on Discord) and Eluxor (Eluxor#8938 on Discord), requested by Reere.

## Why It's Good For The Game

Requested by Reere.

## Changelog
:cl:
add: TE's hardsuit now has a cape module, and TE has a jacket
add: TM's now have access to heat and energy weapon-resistant armor, and helmets
add: Bomj coat added to loot and loadout
imageadd: Added new TE and TM jumpsuit sprites from Armolitskiy and Eluxor (req. by Reere)
imagedel: Removed old TE and TM jumpsuit icons
code: Hardsuit modules now can appear on your sprite if module is set to
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/armor.dm [hunk 1]
```diff
@@ -63,6 +63,21 @@
 	desc = "Aegis Security gear. Protects the head from impacts."
 	icon_state = "helmet_ironhammer"
 
+/obj/item/clothing/head/armor/helmet/technomancer
+	name = "technomancer helmet"
```
Comment (EvilJackCarver) [inline code/modules/clothing/head/armor.dm:67]: Mention of removed lore

File: code/modules/clothing/spacesuits/rig/modules/utility.dm [hunk 2]
```diff
@@ -445,3 +445,13 @@
...
 /obj/item/rig_module/autodoc/commercial
 	autodoc_type = /datum/autodoc/capitalist_autodoc
+
+/obj/item/rig_module/cape
+	name = "cape"
+	desc = "A cape designed to be attached to hardsuits."
+
+/obj/item/rig_module/cape/te
+	name = "technomancer cape"
+	desc = "A tough regal cape, imprinted with the emblem of the Technomancer League."
```
Comment (EvilJackCarver) [inline code/modules/clothing/spacesuits/rig/modules/utility.dm:455]: Mention of removed lore

File: code/modules/clothing/spacesuits/void/station.dm [hunk 3]
```diff
@@ -3,8 +3,8 @@
 /obj/item/clothing/head/space/void/engineering
 	name = "Technomancer voidsuit helmet"
```
Comment (EvilJackCarver) [inline code/modules/clothing/spacesuits/void/station.dm:4]: Something we need to look into

File: code/modules/clothing/suits/jobs.dm [hunk 4]
```diff
@@ -165,6 +165,24 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 	price_tag = 50
 
+//Chief Engineer/Technomancer Exultant
+/obj/item/clothing/suit/storage/te_coat
+	name = "exultant coat"
```
Comment (EvilJackCarver) [inline code/modules/clothing/suits/jobs.dm:170]: lore conflicts again

File: code/modules/clothing/under/jobs/engineering.dm [hunk 5]
```diff
@@ -1,7 +1,12 @@
 //Contains: Engineering department jumpsuits
 /obj/item/clothing/under/rank/exultant
+<<<<<<< HEAD
 	desc = "It's a blue-green jumpsuit worn by engineers. It has minor radiation shielding."
 	name = "chief engineer's jumpsuit"
+=======
+	desc = "It's a yellow-crimson jumpsuit worn by engineers. It has minor radiation shielding."
+	name = "exultant jumpsuit"
+>>>>>>> c53cffa... Adds Armolitskiy's and Eluxor's sprites and items at Reere's Request (#5019)
```
Comment (EvilJackCarver) [inline code/modules/clothing/under/jobs/engineering.dm:9]: Merge conflict.

File: code/modules/clothing/spacesuits/void/station.dm (hunk 3 above)
Comment (drexample) [reply code/modules/clothing/spacesuits/void/station.dm:4]: Will relay to lore team

File: code/modules/clothing/spacesuits/rig/modules/utility.dm (hunk 2 above)
Comment (drexample) [reply code/modules/clothing/spacesuits/rig/modules/utility.dm:455]: Will relay to lore team

File: code/modules/clothing/head/armor.dm (hunk 1 above)
Comment (drexample) [reply code/modules/clothing/head/armor.dm:67]: ditto

**Pull request merged**